### PR TITLE
Add missing minus in `SimilarityTransform` docstring

### PR DESCRIPTION
**Pull request by akonsk**
added a missing minus in the homogeneous transformation matrix

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- Descriptive commit messages (see below)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
  `doc/release/release_dev.rst`.
- There is a bot to help automate backporting a PR to an older branch. For
  example, to backport to v0.19.x after merging, add the following in a PR
  comment: `@meeseeksdev backport to v0.19.x`
- To run benchmarks on a PR, add the `run-benchmark` label. To rerun, the label
  can be removed and then added again. The benchmark output can be checked in
  the "Actions" tab.

### CODE REVIEW

**Comment by rum1887** [comment]
Nice catch ! @akonsk

File: skimage/transform/_geometric.py [hunk 1]
```diff
@@ -1293,7 +1293,7 @@ class SimilarityTransform(EuclideanTransform):
 
     where ``s`` is a scale factor and the homogeneous transformation matrix is::
 
-        [[a0  b0  a1]
+        [[a0  -b0  a1]
```
Comment (rum1887) [inline skimage/transform/_geometric.py:1296]:
```suggestion
        [[a0  b0  a1]
```

**Review by rum1887 — CHANGES_REQUESTED** [review]

File: skimage/transform/_geometric.py (hunk 1 above)
Comment (akonsk) [reply skimage/transform/_geometric.py:1296]: in my understanding, the equations are correct and the matrix is not.
the homogenous matrix should be:
[[s\*cos_theta, -s\*sin_theta, t_y],
[s\*sin_theta, s\*cos_theta, t_x],
[0,0,1]]
here:
a0 = s\*cos_theta
b0 = s\*sin_theta
a1 = t_x
b1 = t_y

in the correction you proposed there is a contradiction, as b0=s\*sin_theta=-s\*sin_theta

**Comment by rum1887** [comment]
Nice catch @akonsk !

**Comment by rum1887** [comment]
hi @akonsk , are you an Outreachy applicant?

File: skimage/transform/_geometric.py (hunk 1 above)
Comment (mkcor) [reply skimage/transform/_geometric.py:1296]: Thanks for the edit, @akonsk! All good now, I recognize the notation found, for example, at: https://en.wikipedia.org/wiki/Transformation_matrix#Affine_transformations

**Comment by mkcor** [comment]
@akonsk would you mind including the same docstring fix for class `EuclideanTransform` (line 1257)? Thanks!

File: skimage/transform/_geometric.py [hunk 2]
```diff
@@ -1293,7 +1293,7 @@ class SimilarityTransform(EuclideanTransform):
...
          [b0  a0  b1]
          [0   0    1]]
```
Comment (mkcor) [inline skimage/transform/_geometric.py:1298]:
```suggestion
         [b0  a0   b1]
         [0   0     1]]
```
(to preserve alignment)

**Comment by lagru** [comment]
Thanks @akonsk!

**Pull request merged**